src/style.css

```diff
@@ -1,3 +1,53 @@
 body {
   margin: 0;
+  font-family: 'Roboto', sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You've specified 'Roboto' as the font family, but there's no fallback font. It's good practice to include a generic font family in case 'Roboto' fails to load, like 'sans-serif'.